src/appshell/internal/startupscenario.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "startupscenario.h"
 
+#include "log.h"
 #include "async/async.h"
```

Comment:
I ask you to adding `log.h` at the very end.
This is due to the fact that many defines are defined in `log.h.
There is a possibility that they will override the definitions defined in other files, which we may not immediately notice and can lead to bad consequences.

Reply:
fixed